modules/monitoring/application/views/scripts/list/servicegrid-flipped.phtml

```diff
@@ -9,6 +9,25 @@ if (! $this->compact): ?>
     <div class="sort-controls-container">
         <?= $this->sortBox ?>
     </div>
+    <?php
+    $toggleFilter = Url::fromRequest();
+    if ($toggleFilter->getParam('problems')) {
+        $title = 'Show entire environment';
+        $toggleFilter->remove('problems');
+    } else {
+        $toggleFilter->setParam('problems');
+        $title = 'Show problem context';
+    }
+    ?>
+    <?= $this->qlink(
+        $this->translate($title),
```

Comment:
Translating a variable causes gettext to not identify this as translatable. Please pass a string here directly.